packages/server/billing/helpers/adjustUserCount.ts

```diff
@@ -97,10 +97,16 @@ const addUser = async (orgIds: string[], user: IUser, dataLoader: DataLoaderWork
       (oldOrganizationUser && oldOrganizationUser.newUserUntil) ||
       organization.periodEnd ||
       new Date()
-    return new OrganizationUser({orgId, userId, newUserUntil, tier: organization.tier})
+    return new OrganizationUser({
+      id: oldOrganizationUser?.id,
```

Comment:
If there is an existing organization user, we overwrite it. Previously we would just insert additional ones.